<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorAnim</class>
 <widget class="QWidget" name="EditorAnim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="TimelineGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QWidget" name="controlWidget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QToolButton" name="removeGraphBtn">
          <property name="toolTip">
           <string>Remove animation.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_unix.qrc">
            <normaloff>:/toolbar/icons/trash-empty.png</normaloff>:/toolbar/icons/trash-empty.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="toolButton_5">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="toolButton_6">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QToolButton" name="toolButton_14">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="quitBtn">
          <property name="toolTip">
           <string>Quit the editor.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources_unix.qrc">
            <normaloff>:/toolbar/icons/system-shutdown.png</normaloff>:/toolbar/icons/system-shutdown.png</iconset>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="toolButton_8">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QToolButton" name="toolButton_13">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="toolButton_11">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="toolButton_12">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="toolButton_7">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="toolButton_9">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="toolButton_10">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimelineGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>timeline/timelinegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources_unix.qrc"/>
 </resources>
 <connections/>
</ui>
